### PR TITLE
Created Hotwax logo component (#1xr7yz3)

### PR DESCRIPTION
**Pull request by azkyakhan**
https://user-images.githubusercontent.com/82817616/150821059-3c5f7c0d-009d-4cde-9bdc-7ab38c352d5a.mp4

### CODE REVIEW

File: src/components/HotwaxLogo.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  name: "HotwaxLogo",
+})
+</script>
+
+<style scoped>
+.dark {
+  display: none;
+}
+
+.light {
```
Comment (dt2patel) [inline src/components/HotwaxLogo.vue:21]: we don't need this class rule right?

File: src/components/HotwaxLogo.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <img class="light" src="../assets/images/hc.png"/>
+    <img class="dark" src="../assets/images/HWCLogoDarkMode.png"/>
+    </figure>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: "HotwaxLogo",
```
Comment (dt2patel) [inline src/components/HotwaxLogo.vue:12]: Lets change this to a generic name like "Logo" so you can put any logo here.

File: src/components/HotwaxLogo.vue [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <figure>
+    <img class="light" src="../assets/images/hc.png"/>
+    <img class="dark" src="../assets/images/HWCLogoDarkMode.png"/>
+    </figure>
```
Comment (dt2patel) [inline src/components/HotwaxLogo.vue:5]: indentation

**Review by dt2patel — CHANGES_REQUESTED** [review]
Some changes, also can you please replace the Hotwax logo everywhere in the SDK so that it is updated for future apps.

File: src/components/HotwaxLogo.vue (hunk 1 above)
Comment (azkyakhan) [reply src/components/HotwaxLogo.vue]: Removed it

File: src/components/HotwaxLogo.vue (hunk 2 above)
Comment (azkyakhan) [reply src/components/HotwaxLogo.vue]: Done

File: src/components/HotwaxLogo.vue (hunk 3 above)
Comment (azkyakhan) [reply src/components/HotwaxLogo.vue]: Done

**Review by dt2patel — APPROVED** [review]

**Pull request merged**